frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -266,6 +266,8 @@ const TextField = ({
           onChange(e as React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>)
         }
         defaultValue={{ id: defaultVal, label: defaultVal }}
+        // the input's text is selected on focus. It helps the user clear the selected value.
+        selectOnFocus
```

Comment:
slightly helps with the issue of the input value not clearing from input field.

![Untitled](https://user-images.githubusercontent.com/39421794/152589007-dd2f4bb9-2ffa-467b-9a05-c8c4e7eac3fe.gif)